internal/cli/enterprise_hooks_scrub.go

```
@@ -0,0 +1,872 @@
+
+func scrubCursorFile(path string, markers []string) (bool, error) {
+	data, err := os.ReadFile(path)
+	if err != nil {
+		return false, err
+	}
+	var cfg map[string]any
+	if err := json.Unmarshal(data, &cfg); err != nil {
+		return false, fmt.Errorf("parse JSON: %w", err)
+	}
+	hooks, _ := cfg["hooks"].(map[string]any)
+	if hooks != nil {
```

golangci-lint (2.12.2)

[error] S1031: unnecessary nil check around range

(staticcheck)